drivers/tty/serdev/pibridge_trace.h

```diff
@@ -220,19 +366,37 @@ DECLARE_EVENT_CLASS(pibridge_timeout_class,
 	)
 );
 
+/*
+ * pibridge_receive_timeout
+ *
+ * Info: timout that occurred while waiting for data reception.
+ * Time: after the timeout occurred.
+ */
 DEFINE_EVENT(pibridge_timeout_class, pibridge_receive_timeout,
 	TP_PROTO(unsigned int received, unsigned int expected,
 		 unsigned int timeout),
 	TP_ARGS(received, expected, timeout)
 );
 
+/*
+ * pibridge_discard_timeout
+ *
+ * Info: timout that occurred while waiting for discard of data.
```

Comment:
A bit more info is probably merited here because the reason why we're waiting for data to be discarded may be non-obvious for an uninitiated reader. How about:

"while receiving a larger than expected packet, a timeout occurred waiting for the excess data (which was going to be discarded)"